ta/pkcs11/src/processing_asymm.c

```diff
@@ -380,6 +380,7 @@ enum pkcs11_rc init_asymm_operation(struct pkcs11_session *session,
 {
 	enum pkcs11_rc rc = PKCS11_CKR_GENERAL_ERROR;
 
+	assert(obj);
```

Comment:
Calling with `obj` being `NULL` is a programming error and is not expected. In such case, core would already catch the issue with a segfault. I don't think the assertion adds much value, but maybe to satisfy sanity source code tools. Otherwise, should all these input argument pointers be tested against `NULL` value?

Reply:
On closer inspection I see now that the logic in `entry_processing_init()` probably keeps `obj` not being NULL in sync with `processing_is_tee_digest()`. The switch between `init_symm_operation`/`init_asymm_operation`/`init_digest_operation` in `entry_processing_init()` was my main concern. Hence I can drop this commit.